homework/generate-sequence/generateSequence.hpp

```diff
@@ -3,5 +3,13 @@
 
 std::vector<int> generateSequence(int count, int step) {
     // TODO: Implement me :)
-    return {};
+    std::vector<int> vec {step};
+    if(count <= 0){
+        return {};
+    }
```

Comment:
Try to figure it out, whether you need this condition? Check what you have in `for` loop.